lib/main_common.pm

```diff
@@ -2400,117 +2400,98 @@ sub load_virtualization_tests {
+        if (get_var('AUTOYAST')) {
+            loadtest "autoyast/installation";
+        } else {
+            loadtest "installation/welcome";
+            loadtest "installation/scc_registration";
+            loadtest "installation/addon_products_sle";
+            loadtest "installation/system_role";
+            loadtest "installation/partitioning";
+            loadtest "installation/partitioning_firstdisk";
+            loadtest "installation/partitioning_finish";
+            loadtest "installation/installer_timezone";
+            loadtest "installation/user_settings";
+            loadtest "installation/user_settings_root";
+            loadtest "installation/resolve_dependency_issues";
+            loadtest "installation/installation_overview";
+            loadtest "installation/disable_grub_timeout";
+            loadtest "installation/start_install";
+            loadtest "installation/await_install";
+            loadtest "installation/reboot_after_installation";
+        }
     }
 
-    if (check_var('VIRT_PART', 'final')) {
-        loadtest 'virtualization/universal/ssh_final';    # Check that every guest is reachable over SSH
-        loadtest 'virtualization/universal/virtmanager_final';    # Check that every guest shows the login screen
-        loadtest "virtualization/universal/smoketest";    # Virtualization smoke test for hypervisor
-        loadtest "virtualization/universal/stresstest";    # Perform stress tests on the guests
-        loadtest "console/perf";
-        loadtest "console/oprofile" unless (get_var("REGRESSION", '') =~ /xen/);
-    }
+    if (check_var('ENABLE_VM_INSTALL', 1)) {
+        loadtest "virt_autotest/login_console";
+        loadtest "virtualization/universal/prepare_guests";
+        loadtest "virtualization/universal/waitfor_guests";
+        if (check_var('PATCH_WITH_ZYPPER', 1)) {
+            loadtest "virtualization/universal/patch_and_reboot";
+            if (my $update_package = get_var('UPDATE_PACKAGE')) {
+                if ($update_package eq 'kernel-default') {
+                    loadtest "virt_autotest/login_console";
+                    loadtest "virtualization/universal/list_guests";
+                    loadtest "virtualization/universal/patch_guests";
+                } elsif ($update_package eq 'xen' || $update_package eq 'qemu') {
+                    loadtest "virt_autotest/login_console";
+                    loadtest "virtualization/universal/list_guests";
+                }
+            }
+        }
+        loadtest "virtualization/universal/kernel";
+        loadtest "virtualization/universal/finish";
+    }
+
+    if (get_var('ENABLE_SRIOV_NETWORK_CARD_PCI_PASSTHROUGH') && is_sle('>15')) {
+        loadtest "console/login";
+        loadtest "virt_autotest/cleanup_libvirtd_log";
+        loadtest "virt_autotest/sriov_network_card_pci_passthrough";
+        loadtest "virtualization/universal/finish";
+    }
+    my @test_cases = (
```

Comment:
Loading tests via this array seems a little bit strange at first glance, but better after looking more. If this is the final choice, suggest to update the var name a little bit -- it makes people feel that earlier loaded tests are not tests ;-)

Reply:
all right, will make changes for var name.